browser/brave_vpn/win/brave_vpn_wireguard_service/status_tray/status_tray_runner.cc

```diff
@@ -118,7 +118,9 @@ void StatusTrayRunner::ConnectVPN() {
   if (IsWireguardActive()) {
     wireguard::EnableBraveVpnWireguardService(
         // passing empty params will reconnect using last known good config.
-        "", "", "", "",
+        // TODO(bsclifton): fetch actual server details instead of relying on
```

Comment:
Some more context: when you install Brave on Windows (system level install) and you enabled VPN, the VPN service is installed which includes registering the tray program with Windows. 

If VPN happens to be disabled, you can right click the Brave VPN tray icon and pick reconnect. The tray runner program will then connect to WireGuard or IKEv2 (WG the default; but tray program works with either) and will initiate a VPN connection with the last used config.

The TODO was added here because it would be nice if the tray runner service had more information about the VPN service / supported servers / existing configs / etc. At the moment, the tray program can only basically disconnect or reconnect.

Reply:
Fixed! Thanks 😄